docs/supported.md

```diff
@@ -393,9 +393,11 @@ Direct Input: $+ - / * ⋅ ± × ÷ ∓ ∔ ∧ ∨ ∩ ∪ ≀ ⊎ ⊓ ⊔ ⊕
 | $\cosec$ `\cosec`  | $\hom$ `\hom`  | $\tg$ `\tg`  | $\Pr$ `\Pr`  |
 | $\cosh$ `\cosh`| $\ker$ `\ker`  | $\th$ `\th`  | $\sup$ `\sup` |
 | $\cot$ `\cot` | $\lg$ `\lg`| $\argmax$ `\argmax` | $\argmin$ `\argmin` |
-| $\plim$ `\plim` | $\operatorname{f}$ `\operatorname{f}`| $\operatorname*{f}$ `\operatorname*{f}`| |
+| $\injlim$ `\injlim` | $\projlim$ `\projlim` | $\varlimsup$ `\varlimsup` | $\varliminf$ `\varliminf` |
+| $\plim$ `\plim` | $\varinjlim$ `\varinjlim` | $\varprojlim$ `\varprojlim` | | 
+| $\operatorname{f}$ `\operatorname{f}`| $\operatorname*{f}$ `\operatorname*{f}`| | |
 
-Functions on the right column of this table can take `\limits`.
+Functions in the right column and bottom three rows of this table can take `\limits`.
```

Comment:
Can `\argmax` take limits as well?.  Maybe an asterisk or something to indicate which items can take `\limits` would be easier to maintain.

Reply:
You know, I should review this table and that statement about the right hand column. I wrote that statement many revisions ago and I am no longer positive that it is still true. I'll spend some time with it tomorrow.

I once considered asterisks, but that would be confusing since asterisks are a part of some LaTeX functions, like `\operatorname*`. So  far, I have not thought of any better way to convey the `limits` idea.

Reply:
I'll mark this PR as a draft for further changes.

Reply:
I have revised the Math Operators table in the documentation. This PR is once again ready for review.